protected/humhub/modules/comment/resources/js/humhub.comment.js

```diff
@@ -251,36 +251,52 @@ humhub.module('comment', function (module, require, $) {
         });
     };
 
-    var toggleComment = function(evt) {
+    var toggleComment = function (evt) {
         var visible = evt.$target.is(':visible');
+        var userName = '@' + evt.$target.closest('.media').find('.media-heading a').html() + ' ';
+        var $form;
+        var target;
+
+        function focusOnForm() {
```

Comment:
Did you miss the parameters here?

Reply:
Thank's for your comment, but no **userName** variable is global in this context... did not need to pass parameter